examples/hacker_news_assets/dbt_project/.gitignore

```diff
@@ -1,3 +1,2 @@
 dbt_modules/
 logs/
-target/
```

Comment:
Why take this out?

Reply:
talked to @OwenKephart, and confirmed that we're using it as the metadata that we load assets from. so it should be checked in.